runtime/src/main/kotlin/kotlin/native/internal/RuntimeUtils.kt

```diff
@@ -80,6 +80,14 @@ internal fun ThrowIncorrectDereferenceException() {
             "Trying to access top level value not marked as @ThreadLocal or @SharedImmutable from non-main thread")
 }
 
+@ExportForCppRuntime
+internal fun ThrowCannotCloneException(original: Any, blocker: Any): Nothing {
+    val originalName = original::class.qualifiedName ?: original::class.simpleName ?: "<object>"
+    val blockerName = blocker::class.qualifiedName ?: blocker::class.simpleName ?: "<object>"
+    throw IllegalArgumentException(
+            "Error trying to clone $original ($originalName) because $blocker ($blockerName) cannot be cloned")
```

Comment:
The operation applied to `original` is not `clone`, it is (currently) `toFrozen`.
I believe the exception message should reflect that.